### PR TITLE
#381 Palette icon positioning does not match cursor when browser zoom…

### PR DESCRIPTION
**Pull request by tomlyn**
… in/out

Fixes: #381 

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by caritaou** [comment]
@tomlyn Your changes fixed the issue reported but im seeing a different behavior where the position of the cursor on the icon being dragged is not centered. Because of this, the location I drop the node onto the palette is not the location where the node appears on the canvas

palette icon cursor

File: canvas_modules/common-canvas/src/common-canvas/svg-canvas-renderer.js [hunk 1]
```diff
@@ -797,26 +802,30 @@ export default class SVGCanvasRenderer {
...
+		const nodeLayout = this.objectModel.getNodeLayout();
+		const ghostAreaMaxWidth = Math.max(nodeLayout.labelMaxWidth, node.width);
 
 		// First remove any old SVG object from the div
 		ghostDivSel
 			.selectAll(".d3-ghost-svg")
 			.remove();
 
-		const ghostGrp = ghostDivSel
+		// Cretae a new SVG area for the ghost area.
```
Comment (caritaou) [inline canvas_modules/common-canvas/src/common-canvas/svg-canvas-renderer.js:820]: typo in the comment

Comment (tomlyn) [reply canvas_modules/common-canvas/src/common-canvas/svg-canvas-renderer.js]: Good catch -- fixed

**Comment by caritaou** [comment]
Sorry for the confusion, please ignore the previous comment. I repulled the code and its working as expected

**Review by caritaou — APPROVED** [review]

**Pull request merged**